routes/index.js

```diff
@@ -2,28 +2,49 @@ var express = require('express');
 var path = require('path');
 var routes = {};
 var path = require('path');
+var passport = require('passport');
 
-var User = require(path.join(__dirname,'../models/studentDataModel')).user;
+var User = require(path.join(__dirname,'../models/user'));
 var Attendence = require(path.join(__dirname,'../models/studentDataModel')).attendence;
 var Data = require(path.join(__dirname,'../models/studentDataModel')).data;
 var Student = require(path.join(__dirname,'../models/studentDataModel')).student;
 var Grades = require(path.join(__dirname,'../models/studentDataModel')).grades;
 
-routes.GEThome = function(req, res, next) {
-  res.sendFile('index.html', { root: path.join(__dirname, '../views') });
-}
-
 routes.POSTlogin = function(req, res, next) {
-  res.sendFile('index.html', { root: path.join(__dirname, '../views') });
+  passport.authenticate('local', function(err, user, info) {
+    if (err) { return res.status(500).send(err.message); }
+    if (!user) {
+      return res.status(401)
+      .send(info.message);}
+    req.logIn(user, function(err) {
+      if (err) { return next(err); }
+      return res.send({redirect: '/'});
+    });
+  })(req, res, next);
 }
 
 routes.POSTregister = function(req, res, next) {
-  res.sendFile('index.html', { root: path.join(__dirname, '../views') });
+  User.register(new User({ email: req.body.username }), req.body.password,
+                function(err, user) {
+    if (err) {
+      console.log('error while user register!', err); return next(err);
+    }
+    console.log('user registered!');
+
+    passport.authenticate('local')(req, res, function () {
+      res.send({redirect: '/'});
+    });
+  });
 }
 
 routes.GETindex = function(req, res){
-	// Student.find
-	// res.JSON({students: allStudents});
+  // Student.find
```

Comment:
should add something in GETindex to redirect you to `/login` if you aren't logged in